src/Microsoft.ML.LightGbm/WrappedLightGbmBooster.cs

```diff
@@ -174,13 +174,13 @@ private static bool FindInBitset(UInt32[] bits, int start, int end, int pos)
         private static int[] GetCatThresholds(UInt32[] catThreshold, int lowerBound, int upperBound)
```

Comment:
>GetCatThresholds [](start = 29, length = 16)

[https://github.com/Microsoft/LightGBM/blob/babe8b313aad747899a3199fba7bc3eae10e582a/src/io/tree.cpp#L270-L279](https://github.com/Microsoft/LightGBM/blob/babe8b313aad747899a3199fba7bc3eae10e582a/src/io/tree.cpp#L270-L279)
This is closest I could find in LightGBM to our logic.